### DISCUSSION TITLE
关于53: read: connection refused的错误提示寻求指导

### DISCUSSION THREAD

**Discussion by lhylvsea** [AlistGo/alist#4406] [Q&A]
在openwrt中进行alist配置阿里云盘后提示这个：
Failed init storage: Post "https://openapi.aliyundrive.com/adrive/v1.0/user/getDriveInfo": dial tcp: lookup openapi.aliyundrive.com on *********1:53: read udp *********:50839->*********1:53: read: connection refused
令牌之类的检查了都没问题
![image_2023-05-17_21-58-41](https://github.com/alist-org/alist/assets/90668922/928b3c49-ab4d-4694-88e0-9abaa0c8bca4)
![image_2023-05-17_21-58-57](https://github.com/alist-org/alist/assets/90668922/6489ff9b-65cb-48e9-9e3e-ca43bbe671ec)
![image](https://github.com/alist-org/alist/assets/90668922/5f99ec5e-6e1b-45fd-9f61-781339eaaa81)

**Comment by lhylvsea** [comment]
补充下，进行123pan或其他的网盘配置后也是这个类似的错误提示。

**Comment by benzeneringlq** [comment]
我在服务器上也出现了你这样，发现是dns解析服务没运行，运行dns解析服务后可以了。你可以`ping openapi.aliyundrive.com`测试
